core/arch/arch.c

```diff
@@ -3254,7 +3254,14 @@ check_syscall_method(dcontext_t *dcontext, instr_t *instr)
         ASSERT(get_syscall_method() == SYSCALL_METHOD_UNINITIALIZED ||
                get_syscall_method() == SYSCALL_METHOD_INT);
 #    ifdef LINUX
-        if (new_method == SYSCALL_METHOD_SYSENTER) {
+
+        /* PR 5037: 32-bit apps on AMD machines have OP_syscall instruction in
```

Comment:
I will take this into account for the next collaboration.

It is strange, my repo is public.
Maybe the button grayed out because I not synced with upstream. Already I have done it. Please try again.

If the button will still inactive then you can send me the text and I will change it.

Reply:
"Edit file" is still grayed out.  Please change this comment to something like:

```C
/* i#4407: An OP_syscall instruction on 32-bit AMD returns to a hardcoded vsyscall PC no
 * matter where it is.  Thus we must hook the vsyscall just like we do for OP_sysenter.
 */
```

Reply:
Changed